cmd/vic-machine/main.go

```diff
@@ -139,6 +140,12 @@ func main() {
 		logs = append(logs, f)
 	}
 
+	// create VCH logger for streaming creation logs to datastore folder
```

Comment:
It seems like the logging framework we're using, logrus, [uses hooks as an extensibility model](https://github.com/sirupsen/logrus#hooks). Could our new logging functionality be simplified by following that pattern?

Reply:
hook is essentially callback function that gets called every time a log message is written. `Datastore.Upload` provided by govmomi overwrites the original content, so we can't do upload in hook (every `log.infof`, `log.debugf` and the like will be followed with a upload, and the previous log message got overwritten by the next one, etc.)
Also, we know exactly what point of time the datastore folder is ready, so there's no need to do signaling work in callback hook.